wrangler-core/src/test/java/io/cdap/wrangler/parser/RecipeCompilerTest.java

```diff
@@ -183,26 +182,31 @@ public void test() throws Exception {
       "send-to-error body_5 =~ \"DC.*\"",
       "filter-rows-on regex-match body_5 *as*"
     };
-    CompileStatus compile = TestingRig.compile(recipe);
-    Assert.assertTrue(true);
+    TestingRig.compileSuccess(recipe);
   }
 
   @Test
   public void testSingleLineDirectives() throws Exception {
     String[] recipe = new String[] {
       "parse-as-csv :body '\t' true; drop :body;"
     };
-    CompileStatus compile = TestingRig.compile(recipe);
-    Assert.assertTrue(true);
+    TestingRig.compileSuccess(recipe);
+  }
+
+  @Test
+  public void testHandlingCRLF() throws Exception {
+    String[] recipe = new String[] {
+      "parse-as-csv :body '\t' true;\r\ndrop :body;\r\n"
+    };
+    TestingRig.compileSuccess(recipe);
```

Comment:
shouldn't just test that it compiles (didn't it compile successfully before?) Should test that the directives are parsed into the exact strings that are expected.